ex02.exs

```diff
@@ -1,6 +1,24 @@
 
 defmodule Ex02 do
 
+  @name __MODULE__
+
+  def new_counter(value \\ 0) do
+    { :ok, _ } = Agent.start_link(fn -> value end)
```

Comment:
## -2

From a style point of view, it's better not to be passing the `{:ok, pid}` tuple around everywhere. The `:ok` part is only really used once, to validate the call the `start_link` worked. After that it can be safely discarded.

But…